cmd/serverless-init/main.go

```diff
@@ -32,6 +32,10 @@ func main() {
 		panic("[datadog init process] invalid argument count, did you forget to set CMD ?")
 	}
 
+	// The datadog-agent requires Load to be called or it could
+	// panic down the line.
+	_, _ = config.Load()
```

Comment:
maybe we should check for the error like we did in the extension? 
https://github.com/DataDog/datadog-agent/blob/main/cmd/serverless/main.go#L211

also do you mind adding a unit test for that to avoid regression? maybe with creating a dummy datadog.yml file with one config, and then checking that specific config is loaded via `config.Datadog.getString()`?

Reply:
from what I can tell I think this function **always** errors in serverless-init. Let me try and figure out why. I was hopeful we could just call it here and ignore the error as we don't really need any of the functionality, but let me go ahead and confirm.